project/Dependencies.scala

```diff
@@ -29,6 +29,7 @@ object Dependencies {
     "io.circe" %% "circe-core" % "0.14.2",
     "io.circe" %% "circe-generic" % "0.14.2",
     "io.circe" %% "circe-parser" % "0.14.2",
-    "io.circe" %% "circe-generic-extras" % "0.14.2"
+    "io.circe" %% "circe-generic-extras" % "0.14.2",
```

Comment:
We could follow the same conventions as in other projects with respect to how we specify versions of dependencies and so on. Not necessarily as part of this PR, but tickets for this could be created as we go and references left in the code.

Reply:
Unfortunately I am not sure, we have yet a settled standard for this. In any case, I added constants for the versions at least (grouped in its own object).